FordLabs/PeopleMover
# 118 add filter by role

## Issue
Connects #118

## What was done
- [x] Added the ability to filter by role
- [x] Added end to end test for filter by role

## How to test
1. Use the filter down to filter by a specific role
2. Make sure that only people with the specified role show up on products and in the unassigned drawer
